src/test/java/com/google/api/codegen/testdata/java_mock_service_impl_library.baseline

```diff
@@ -23,6 +23,7 @@ import com.google.tagger.v1.AddLabelResponse;
 import com.google.tagger.v1.LabelerGrpc.LabelerImplBase;
 import io.grpc.stub.StreamObserver;
 import java.util.ArrayList;
+import java.util.Arrays;
```

Comment:
Unfortunately no, the formatter only supports java files. This is the drawback of this approach that the baseline files will contain some unnecessary imports.